client/src/components/UserScore.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
This is your scoreboard page, it just needs renaming to ScoreboardPage.js and putting in the pages folder